modules/core/src/main/java/org/apache/ignite/internal/cdc/CdcMain.java

```diff
@@ -204,6 +229,12 @@ public class CdcMain implements Runnable {
     /** Time of last segment consumption. */
     private AtomicLongMetric lastSegmentConsumptionTs;
 
+    /** Metric shows duration of the last WAL segment events consumption. */
```

Comment:
Fix it in the same way as the documentation.

Reply:
Fixed